geoh5py/objects/geo_image.py

```diff
@@ -323,6 +362,8 @@ def georeferencing_from_tiff(self):
             v_cell_size = float(self.tag[33550][1])
             u_count = float(self.tag[256][0])
             v_count = float(self.tag[257][0])
+
+            # todo: not anymore because it's not always a rectangle
```

Comment:
Address now or in another PR?

Reply:
..just checking because sometimes I leave myself a todo comment to come back in the same issue.  If it isn't the case, maybe you can create an issue to address the todo and add the issue number in the comment.

Reply:
later [GEOPY-2591](https://mirageoscience.atlassian.net/browse/GEOPY-2591)